beacon-chain/blockchain/forkchoice/process_block.go

```diff
@@ -381,11 +382,6 @@ func (s *Store) rmStatesBySlots(ctx context.Context, startSlot uint64, endSlot u
 	ctx, span := trace.StartSpan(ctx, "forkchoice.rmStatesBySlots")
 	defer span.End()
 
-	// Do not remove genesis state.
```

Comment:
Why remove this?

Reply:
Yeah. You are right. I should have it back as 2nd layer check